content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -199,6 +202,56 @@ available for the log backend:
 
 By default truncate is disabled in both `webhook` and `log`, a cluster administrator should set `audit-log-truncate-enabled` or `audit-webhook-truncate-enabled` to enable the feature.
 
+### Dynamic backend
+
+{{< feature-state for_k8s_version="v1.13" state="alpha" >}}
+
+In Kubernetes version 1.13, you can configure dynamic audit webhook backends AuditSink API objects. 
+
+To enable dynamic auditing you must set the following apiserver flags:
+
+- `--audit-dynamic-configuration`: the primary switch. When the feature is at GA, the only required flag.   
+- `--feature-gates=DynamicAuditing=true`: feature gate at alpha and beta.   
```

Comment:
Please revise content/en/docs/reference/command-line-tools-reference/feature-gates.md by documenting this feature gate

Reply:
#11292